### PR TITLE
docs/KEYS: Fix broken link to keys

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
Quick fix.

### CODE REVIEW

**Comment by surajssd** [comment]
While I was looking at this I got following errors: 

For https://pgp.mit.edu/pks/lookup?search=0x9725A27E0A93A153B798F2928DBCFB00F79374C3&op=index&exact=on

```
Proxy Error
The proxy server received an invalid response from an upstream server.
The proxy server could not handle the request GET /pks/lookup.

Reason: Error reading from remote server

```

And for https://pgp.mit.edu/pks/lookup?search=0xB7DDBE7001408D7D7FD079D27E26368261BAD4F7&op=index&exact=on

```
Timeout
Request timed during request parsing after 5 seconds
```

Will give it a try again.

File: docs/KEYS.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ This is the current set of trusted keys:
 
 | Short fingerprint               | Full fingerprint                                      | Owner         |
 | ------------------------------- | ----------------------------------------------------- | ------------- |
-| [0x61BAD4F7](key-iaguis)        | `B7DD BE70 0140 8D7D 7FD0 79D2 7E26 3682 61BA D4F7`   | @iaguis       |
```
[the remark this answers is not in the document]
Comment (invidian) [reply docs/KEYS.md:10]: Following markdown works for me:
```
| [0x61BAD4F7][key-iaguis]        | `B7DD BE70 0140 8D7D 7FD0 79D2 7E26 3682 61BA D4F7`   | @iaguis       |

[key-iaguis]: https://pgp.mit.edu/pks/lookup?search=0xB7DDBE7001408D7D7FD079D27E26368261BAD4F7&op=index&exact=on
```

**Comment by invidian** [comment]
>Will give it a try again.

@surajssd this is just https://pgp.mit.edu being unavailable, which is the case most of the time in my experience. But this is not relevant to this PR, this PR is about bad markdown syntax for links.

File: docs/KEYS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mauriciovasquezbernal) [reply docs/KEYS.md:10]: Oh right, I didn't know about that.
Fixed.

**Comment by invidian** [comment]
Thanks @mauriciovasquezbernal!

**Pull request merged**